### ISSUE TITLE
[FEATURE REQUEST] Brief description

### ISSUE THREAD

**Issue by r4tna** [martinet101/ElevenClock#1032]
In parallel with ElevenClock I run a util to show me if Num lock is pressed (it shows caps and scroll lock too).

If ElevenClock had a num lock indicator, it was be great as I have everything I need in one util.

**Comment by martinet101** [comment]
Hi @r4tna,

I understand what you mean, but this would be out of scope for the current project.

I'm sorry, but this won't be worked on

**Comment by r4tna** [comment]
No worries - was just a suggestion.

You did a great job already!

Thanks,
Ratna
